sandbox/controls/ListItems.qml

```diff
@@ -347,6 +347,17 @@ CExPynn1gWf9bx498P7/nzPcxEzGExhBdJGYihtAYQlO+tUZvqrPbqeudo5iJGEJjCE15a3VtodH3q2I
         requestsCount: 3
     }
 
+    ListModel {
+        id: identiconRing1
```

Comment:
is this model used / going to be used elsewhere? If not, could we assign it directly to the component that's using it?

Reply:
Yes, it could be assigned directly!